src/controller/file_manager.controller.ts

```diff
@@ -64,6 +70,20 @@ export class FileManagerController {
     this.fileManagerService.ensureLocationCanBeUsed(body.location);
 
     const fileSignedPostUrls = await this.fileManagerService.getSignedPostUrls(user, body);
-    return this.fileManagerFormatter.toLinksResponse(fileSignedPostUrls);
+    return this.fileManagerFormatter.toLinksResponsePost(fileSignedPostUrls);
+  }
+
+  @Post('/signed-urls/get')
+  @HttpCode(HttpStatus.OK)
+  @ApiBearerAuth('authorization')
+  @ApiResponse({ status: HttpStatus.OK, type: SignedGetUrlsResponse })
+  public async getSignedGetUrls(
```

Comment:
Can you please change HTTP method from POST to GET, and the same for getSignedPostUrls. And from POST to DELETE for delete request

Reply:
Done